.test.yml

```diff
@@ -0,0 +1,15 @@
+- name: verify
```

Comment:
Will you please add a comment to the top of the file explaining it? This is important since it is the first version of this file, and everyone who makes their own will tend to just copy this one.

IE: explain what this file does, its format including the purpose of each tag.